ppr-ui/src/components/tombstone/TombstoneDischarge.vue

```diff
@@ -94,7 +97,11 @@ export default defineComponent({
         return 'No Expiry'
       }),
       statusType: computed((): string => {
-        return getMhrInformation.value?.statusType || 'N/A'
+        const parentReg = getMhrInformation.value.mhrNumber &&
+                          getMhRegTableBaseRegs.value?.find(reg => reg.mhrNumber === getMhrInformation.value.mhrNumber)
+        const status = parentReg?.statusType
+        if (status === mhApiStatusTypes.FROZEN) return 'Active'
```

Comment:
Based on line 102, `parentReg` is optional, so const `status` could be undefined. You should add another check.

Reply:
Honestly, i think you can just simply map `FROZEN` to `Active`.
As Doug was saying yesterday, its a sub state of an `ACTIVE` registration, so not much risk there. 

ie return `getMhrInformation.value?.statusType === mhApiStatusStype.FROZEN ? 'mhUiStatusTypes.ACTIVE : getMhrInformation.value?.statusType || 'N/A'`

Reply:
Even better!

Reply:
@cameron-eyds simplified as much as possible, still need to depend on parent statusTypes for say a draft on a historical/exempt registration

Reply:
It looks like you rebased! 
So, in that case, you can pull `isFrozenMhr` computed value from the `useMhrInformation` composable. 

This handles the draft scenario 👍